MapleServer2/Database/Classes/DatabaseItem.cs

```diff
@@ -185,7 +186,8 @@ public void Update(Item item)
             transfer_flag = item.TransferFlag,
             transparency_badge_bools = JsonConvert.SerializeObject(item.TransparencyBadgeBools),
             unlock_time = item.UnlockTime,
-            ugc_uid = item.Ugc == null ? null : (int?) item.Ugc.Uid
+            ugc_uid = item.Ugc == null ? null : (int?) item.Ugc.Uid,
+            pet_info = JsonConvert.SerializeObject(item.PetInfo)
```

Comment:
Pets seem like they're going to become length with all their stats, settings, etc. Maybe separate this out into it's own table?

Reply:
Stats are handled already by Item, but sure i can add these pet info to another table